pyproject.toml

```diff
@@ -71,6 +71,9 @@ pyrender = {git = "https://github.com/mmatl/pyrender.git", markers = "sys_platfo
 hello-robot-stretch-body = {version = ">=0.7.27", markers = "sys_platform == 'linux'", optional = true}
 pyserial = {version = ">=3.5", optional = true}
 jsonlines = ">=4.0.0"
+transformers = "^4.47"
+torchmetrics = "^1.6.0"
```

Comment:
Added to track default models metrices, I have used them to check the reward classifier.